### PR TITLE
Changes for migration on zephyr-v3.3.0-xt branch

### PR DESCRIPTION
**Pull request by firscity**
Changes, needed for main branch to make it buildable with new zephyr-v3.3.0-xt branch of Zephyr OS.

### CODE REVIEW

**Review by xakep-amatop — APPROVED** [review]
`Acked-by: Mykola Kvach <mykola_kvach@epam.com>`

**Comment by LKomaryanskiy** [comment]
`Acked-by: Leonid Komarianskyi <leonid_komarianskyi@epam.com>`

**Comment by dsemenets** [comment]
`Reviewed-by: Dmytro Semenets<dmytro_semenets@epam.com>`

File: xen-dom-mgmt/include/uimage.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ */
+#ifndef __XENLIB_XEN_UIMAGE_H__
+#define __XENLIB_XEN_UIMAGE_H__
+
+#include <zephyr/kernel.h>
+
+#define UIMAGE_MAGIC 0x27051956
+#define UIMAGE_NMLEN 32
+
+struct uimage_hdr {
```
Comment (lorc) [inline xen-dom-mgmt/include/uimage.h:15]: There is no guarantee that contents of this structure will be aligned with actual header. `packed` attribute might help a bit

File: xen-dom-mgmt/include/zimage.h [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#ifndef __XENLIB_XEN_ZIMAGE_H__
+#define __XENLIB_XEN_ZIMAGE_H__
+
+#include <zephyr/kernel.h>
+
+#define ZIMAGE64_MAGIC_V0	0x14000008
+/* ASCII ARM\x64 */
+#define ZIMAGE64_MAGIC_V1	0x644d5241
+
+struct zimage64_hdr {
```
Comment (lorc) [inline xen-dom-mgmt/include/zimage.h:16]: There is no guarantee that contents of this structure will be aligned with actual header. `packed` attribute might help a bit

**Review by lorc — APPROVED** [review]
With comments fixed:

`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

File: xen-dom-mgmt/include/uimage.h (hunk 1 above)
Comment (firscity) [reply xen-dom-mgmt/include/uimage.h:15]: Good point, thanks. Fixed.

**Review by sa-kib — APPROVED** [review]
LG, +1

**Pull request merged**